### ISSUE TITLE
Mana Spiral calculations at 64MC

### ISSUE THREAD

**Issue by ascriptmaster** [junlico/dragalia-lost#19]
For people who do not have enough spheres from certain High Dragon Trials (cough Jupiter cough) it would be useful to have a setting for Mana Spiral 64MC, which would cap level at 95 and ignore the extra boosts from the last few nodes to ensure the correct calculations for opening blast.

**Comment by junlico** [comment]
There's only a whole level value (_AddMaxHp1 & _AddMaxAtk1) in the raw chara data.
We might get that info if parse deeply into mana data, but that's too complicated for now.
PR always welcome :)

**Comment by ascriptmaster** [comment]
What do you mean by this? Isn't mana spiral node information publicly available on Gamepedia and the like? Or is there something I'm missing?

**Comment by junlico** [comment]
Gamepedia data is also parsed from the raw data from game app. That's the effort of multiple volunteers.

We can get whole level mana circle value directly from character's data (only parse one file), but if want single node, I need parse deeply from other files to fit data model.

TL;DR
 too lazy to do that.

**Comment by ascriptmaster** [comment]
Hmmm. I was basically just thinking 64 specifically because of the no-tails use case, but if that's too difficult, that's fine.
